Java/CosmosDB-NoSQL-RAG-Chatbot/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-chatgpt-sample-cosmos</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>spring-chatgpt-sample-cosmos</name>
+    <description>Demo project for Spring Boot</description>
+    <packaging>pom</packaging>
+    <modules>
+        <module>spring-chatgpt-sample-common</module>
+        <module>spring-chatgpt-sample-webapi</module>
+        <module>spring-chatgpt-sample-cli</module>
+    </modules>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.0.6</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <properties>
+        <java.version>17</java.version>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-spring-data-cosmos</artifactId>
+            <version>3.46.0</version>
```

Comment:
Is there a way to have this version automatically update to the latest (maybe it already does) or with samples do we have to go in and manually update this when new versions are released? I believe in our repo this is done with comment block that is used for replacement with the latest, but I'm not 100% on that.

Reply:
Below uses LATEST so I'd just use that